_includes/deep_learning_form.html

```diff
@@ -0,0 +1,61 @@
+<div id="mc_embed_signup">
+  <form
+    action="https://twitter.us14.list-manage.com/subscribe/post?u=75419c71fe0a935e53dfa4a3f&id=91d0dccd39"
+    method="post"
+    id="mc-embedded-subscribe-form"
+    name="mc-embedded-subscribe-form"
+    class="email-subscribe-form validate deep-learning-form"
+    target="_blank">
+    <div id="mc_embed_signup_scroll" class="email-subscribe-form-fields-wrapper col-8">
+      <div class="mc-field-group">
+        <label for="mce-EMAIL" style="display:none;">Email Address</label>
+        <input type="email" value="" name="EMAIL" class="required email deep-learning-input" id="mce-EMAIL" placeholder="Enter Your Email">
+      </div>
+  
+      <div id="mce-responses" class="clear">
+        <div class="response" id="mce-error-response" style="display:none"></div>
+        <div class="response" id="mce-success-response" style="display:none"><a href="{{ site.base}}/deep-learning-with-pytorch-thank-you" target="_blank"></a></div>
+      </div>    <!-- real people should not fill this in and expect good things - do not remove this or risk form bot signups-->
+  
+      <div style="position: absolute; left: -5000px;" aria-hidden="true"><input type="text" name="b_75419c71fe0a935e53dfa4a3f_91d0dccd39" tabindex="-1" value=""></div>
+
+      <div class="deep-learning-button">
+        <button id="deep-learning-button" class="btn btn-lg with-right-arrow module-button email-subscribe-button">
+          Download Book
+        </button>
+      </div>
```

Comment:
I think having a class and ID named the same is confusing. You might not even need the div container anyway. You can take the `margin-top` you're adding to the div and just add it to the button.